app/bt/AffectedUserFlow/PublishConsent.tsx

```diff
@@ -1,38 +1,111 @@
 import { useTranslation } from 'react-i18next';
 import { useNavigation } from '@react-navigation/native';
 
-import ExportTemplate from './ExportTemplate';
+import { Button } from '../../components/Button';
+import { Typography } from '../../components/Typography';
 import { useStrategyContent } from '../../TracingStrategyContext';
-import { Screens } from '../../navigation';
+import { useStatusBarEffect } from '../../navigation';
 import * as BTNativeModule from '../nativeModule';
 
+import { Screens } from '../../navigation';
+import { Images } from '../../assets';
+import {
+  Layout,
+  Colors,
+  Spacing,
+  Iconography,
+  Typography as TypographyStyles,
+} from '../../styles';
+
 const PublishConsent = (): JSX.Element => {
+  useStatusBarEffect('dark-content');
   const navigation = useNavigation();
-  const [isConsenting, setIsConsenting] = useState(false);
+  const [isLoading, setIsLoading] = useState(false);
   const { t } = useTranslation();
 
   const { StrategyCopy, StrategyAssets } = useStrategyContent();
 
-  const consent = async () => {
-    setIsConsenting(true);
+  const handleOnPressConfirm = async () => {
+    setIsLoading(true);
     const cb = (_errorMessage: string, _successMessage: string) => {
       navigation.navigate(Screens.AffectedUserComplete);
-      setIsConsenting(false);
+      setIsLoading(false);
     };
     BTNativeModule.submitDiagnosisKeys(cb);
   };
 
+  const title = StrategyCopy.exportPublishTitle;
+  const body = t('export.publish_consent_body_bluetooth');
+
   return (
-    <ExportTemplate
-      onNext={consent}
-      icon={StrategyAssets.exportPublishIcon}
-      headline={StrategyCopy.exportPublishTitle}
-      body={t('export.publish_consent_body_bluetooth')}
-      nextButtonLabel={t('export.consent_button_title')}
-      buttonLoading={isConsenting}
-    />
+    <ImageBackground
+      source={Images.BlueGradientBackground}
+      style={styles.backgroundImage}>
+      <SafeAreaView style={{ flex: 1 }}>
+        <ScrollView
+          style={{ flex: 1 }}
+          contentContainerStyle={styles.contentContainer}>
+          <View style={styles.icon}>
+            <SvgXml
+              xml={StrategyAssets.exportPublishIcon}
+              width={Iconography.small}
+              height={Iconography.small}
+            />
+          </View>
+
+          <View style={styles.content}>
+            <Typography style={styles.header}>{title}</Typography>
+            <Typography style={styles.contentText}>{body}</Typography>
+          </View>
+
+          <Button
+            label={t('export.consent_button_title')}
+            onPress={handleOnPressConfirm}
+            loading={isLoading}
+          />
+        </ScrollView>
+      </SafeAreaView>
+    </ImageBackground>
   );
 };
 
+const styles = StyleSheet.create({
+  contentContainer: {
+    paddingTop: Layout.oneTenthHeight,
+    paddingHorizontal: Spacing.large,
+    paddingBottom: Spacing.huge,
+  },
+  backgroundImage: {
```

Comment:
Not a fan of extracting code for the sake of it but this style feels like one that will be copied over verbatim a few times, would it make sense to extract it like we did with the `secondaryContent` or `header2` ?

Reply:
It might do, but we where chatting about this while pairing and we think the root issue is the way the navigation is set up for this stack requires the 100% cover background image. It would be preferable to solve the root issue and remove `<ImageBackground/>` altogether.